packages/headless/src/app/navigatorContextProvider.ts

```diff
@@ -33,6 +42,7 @@ export const defaultBrowserNavigatorContextProvider: BrowserNavigatorContextProv
     userAgent: navigator.userAgent,
     location: window.location.href,
     clientId,
+    capture: clientId !== '',
```

Comment:
This seems like a good default value for CSR: by default we'll disable `capture` if `clientId` is an empty string, so requests won't fail.